gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/ai/engine/services/openai/aiengine/aiengine/CohereAiServiceAdapter.java

```diff
@@ -0,0 +1,42 @@
+package dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.services.openai.aiengine.aiengine;
+
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.request.AiModelMessage;
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.request.BotRequest;
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.request.AiModelRequestParams;
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.response.CompletionResponse;
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.response.EmbeddingResponse;
+import dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.response.ModerationResponse;
```

Comment:
The class `ev.ctrlspace.gendox.gendoxcoreapi.ai.engine.model.dtos.openai.response.ModerationResponse;` does not exist